yb-voyager/cmd/import.go

```diff
@@ -354,8 +354,17 @@ func registerFlagsForTarget(cmd *cobra.Command) {
 	BoolVar(cmd.Flags(), &tconf.EnableYBAdaptiveParallelism, "enable-adaptive-parallelism", true,
 		"Adapt parallelism based on the resource usage (CPU, memory) of the target YugabyteDB cluster.")
 	cmd.Flags().IntVar(&tconf.MaxParallelism, "adaptive-parallelism-max", 0,
-		"number of max parallel jobs to use while importing data when adaptive parallelism is enabled."+
+		"number of max parallel jobs to use while importing data when adaptive parallelism is enabled. "+
 			"By default, voyager will try if it can determine the total number of cores N and use N/2 as the max parallel jobs. ")
+	BoolVar(cmd.Flags(), &skipReplicationChecks, "skip-replication-checks", false,
```

Comment:
In the help msg, we should indicate that voyager will abort if it detects replication on the target cluster.

Reply:
How about something like `It is NOT recommended to have any form of replication (CDC/xcluster) running on the YB cluster during import-data. If detected, import-data is aborted. Use this flag to turn off the checks and continue importing data`